src/data/repositories/ConstantD2Repository.ts

```diff
@@ -54,8 +65,6 @@ export class SettingsD2Repository implements SettingsRepository {
                     },
                 ],
             })
-        ).map(() => {
-            return settings;
-        });
+        ).map(() => settings);
```

Comment:
Any metadata post needs its response to be validated. Check `runMetadata` in other projects (i.e. central-planning-reporting)